docs/open-source-tizen/tizen-studio/package-file-desc-guide.md

```diff
@@ -22,7 +22,7 @@ Project structure for developing, packaging, and building extensions requires th
   - `<package name>.remove.<TARGET OS>`: post-removal script
 
 
-### `pkginfo.manifest` File
+### pkginfo.manifest File
```

Comment:
File name has code tag like below:
### `pkginfo.manifest` File

This is one of rules on styleguide.